invenio_app_rdm/theme/webpack.py

```diff
@@ -26,8 +26,25 @@ def theme():
         entry={
             'invenio-app-rdm-theme': current_app.config['INSTANCE_THEME_FILE'],
             'invenio-app-rdm-js': './js/invenio_app_rdm/inveniordm.js',
```

Comment:
Wasn't this file removed? If so, not sure of the implications (should be fine tho).